### PR TITLE
feat: create form farm

### PR DESCRIPTION
**Pull request by nguyenvuces**
add form add farm both create account and farm

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/Code-Engine-Studio/ces-semantic-release/blob/main/README.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by ces-huy-dang — APPROVED** [review]

File: web/modules/custom/maybe_forms/maybe_forms.module [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * @file
+ * Custom Blocks module.
+ */
+
+const MAYBE_FORMS_FARMER_ROLE = 'farmer';
+const MAYBE_FORMS_CONSUMER_ROLE = 'consumer';
+const MAYBE_FORMS_UNIQUE_ERROR_MESSAGE = "Your email has been used.";
+const MAYBE_FORMS_ERROR_MESSAGE = "An error occurred while creating the farm.";
+const MAYBE_FORMS_STORE_TYPE = "online";
+const MAYBE_FORMS_DEFAULT_PASSWORD = "123456";
```
Comment (ces-khoado) [inline web/modules/custom/maybe_forms/maybe_forms.module:13]: Password should be an encoded string of smth (for example: the timestamp when user is created)

File: web/modules/custom/maybe_forms/src/Services/AccountService.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  /**
+   * Checks if a user with the given username exists in the system.
+   *
+   * @param string $username
+   *   The username to check for existence.
+   *
+   * @return bool
+   *   TRUE if a user with the specified username exists, FALSE otherwise.
+   */
+  public function checkUserIsExist($username) {
```
Comment (ces-khoado) [inline web/modules/custom/maybe_forms/src/Services/AccountService.php:55]: Change this function's name, exist is not a noun.

File: web/modules/custom/maybe_forms/maybe_forms.module (hunk 1 above)
Comment (nguyenvuces) [reply web/modules/custom/maybe_forms/maybe_forms.module:13]: In the feature there is no feature to send the user the password, setting the default password is a temporary function in this feature.

File: web/modules/custom/maybe_forms/src/Services/AccountService.php (hunk 2 above)
Comment (nguyenvuces) [reply web/modules/custom/maybe_forms/src/Services/AccountService.php]: updated

**Review by ces-khoado — APPROVED** [review]

**Pull request merged**